Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM node:lts as build-stage
+
+# Install deps
+RUN apt-get update -y \
+    && export DEBIAN_FRONTEND=noninteractive \
+    && apt-get install --no-install-recommends -y wget=1.* \
+    && rm -rf /var/lib/apt/lists/*
+RUN wget --progress=dot:giga -O nvim.appimage \
+  'https://github.com/neovim/neovim/releases/download/v0.4.4/nvim.appimage'
+RUN chmod +x nvim.appimage
+RUN ./nvim.appimage --appimage-extract
+RUN chmod -R a+rx /squashfs-root/
+RUN mkdir -p /usr/local/bin
+RUN ln -s /squashfs-root/AppRun /usr/local/bin/nvim
+
+# Drop privileges and build firenvim
+RUN useradd --create-home --user-group user
+USER user
+COPY --chown=user:user . /firenvim
+WORKDIR /firenvim
+RUN npm install
+RUN npm run build
+RUN npm run install_manifests
+
+FROM scratch AS export-stage
+COPY --from=build-stage /firenvim/target /
```

Comment:
Is the final `/` relative to the folder the Dockerfile is located? Or the PWD at build time? Or something else? This is new syntax to me, I didn't know you could copy out to the host without mounting it as a volume.

Reply:
Relative to a directory you pass when you build (`--output`): `DOCKER_BUILDKIT=1 docker build . -t firenvim --output target`

Reply:
This is a relatively new feature: https://docs.docker.com/develop/develop-images/build_enhancements/